jobs/azure-discovery/src/main/java/com/tmobile/pacbot/azure/inventory/AzureDiscoveryJob.java

```diff
@@ -60,9 +61,11 @@ public static Map<String, Object> execute(Map<String,String> params){
 				params.forEach((k,v) -> System.setProperty(k, v));
 			}
 		} catch (Exception e) {
-			log.error("Error fetching config", e);
+			log.error("Error occurred in job azure-data-collector-job while fetching config", e);
```

Comment:
First, this additional info seems useless because this log will be part of azure-data-collector-job logs.
Second, it will become even less useful is we change the job name.

Reply:
Error occurred in job% --- datadog will read this and create data alert. Data dog will read all error logs and look for this string. name 'azure-data-collector-job' is not important.

Reply:
Can datadog look into where the error happened? Like in which asset that happened?
Like for example we do have monitor for Rule Engine
https://app.datadoghq.com/monitors/130654998?view=spans

Reply:
from my perspective log already contains all info needed: the fact that error happened, exact place where it happened, presumably what exactly happened. can all of this be used by DataDog instead of altering every single log message in every job?

Reply:
If you observe, we don't want to create data alert for all errors. That's why, i put this logger string only in some places which are good enough to create data alert. We don't want to burden data dog by providing several strings, instead look only for one particular string.

Reply:
@mkinchin-paladin, we implemented datadog monitoring based on a particular string from these collectors. 

I got what you are proposing, but we need a spike on datadog to see if it can take the info and create alerts based on the log stream where the error occurred. cc @plyubich @rnithinpaladin